### ISSUE TITLE
Investigate stringiness reports

### ISSUE THREAD

**Issue by vintagepc** [vintagepc/RPM-Post#13]
In #5 it was reported the RPM script seems to leave stringier tips than the wipe tower. 
Look into this and see why.

**Comment by vintagepc** [comment]
@bynnek - I just took a look at this. The ramming E move distances for the script vs the tower are identical, as are the cooling moves. The only compounding issue is that for the wipe tower, it's also moving X/Y in addition to E, which means I can't directly compare the speed F values. However, I did reverse one of the  wipetower X/F calculations and they work out - i.e. the F rates there are indeed such that the travel move should take 0.25 seconds. Marlin does linear interpolation so E arrives at its coordinate at the same time X does.

Long story short - I'm not sure why the RPM would cause stringier tips vs the wipe tower. I'm open to ideas, but unfortunately I don't see much that can be done here apart from adjusting the ramming and cooling settings. (Sorry).
